src/verify_cert.rs

```diff
@@ -985,20 +941,30 @@ mod tests {
         )
     }
 
-    #[test]
-    #[cfg(feature = "alloc")]
-    fn longest_allowed_path() {
-        assert_eq!(build_linear_chain(1), Ok(()));
-        assert_eq!(build_linear_chain(2), Ok(()));
-        assert_eq!(build_linear_chain(3), Ok(()));
-        assert_eq!(build_linear_chain(4), Ok(()));
-        assert_eq!(build_linear_chain(5), Ok(()));
-        assert_eq!(build_linear_chain(6), Ok(()));
+    fn make_issuer(org_name: String) -> rcgen::Certificate {
```

Comment:
Suggestion: have it take `impl Into<String>`?

Reply:
Done, thanks.